assets/css/main.css

```
@@ -1116,3 +1116,65 @@ ul.post-archive{margin-left:0!important;}
 .error-wrap .message-manual {
     margin-bottom: 32px;
 }
+
+/* ====================== bookmark card ====================== 2019.09.07 */
+.kg-bookmark-container {
+	display: flex;
+	flex-wrap: wrap;
+	color: black;
+	text-decoration: none;
+	margin: 0;
+	border-radius: 5px;
+	overflow: hidden;
+	border: 1px solid #e5eff5 !important;
+	box-shadow: 0 0 1px rgba(0,0,0,.06),0 2px 6px rgba(0,0,0,.03);
+}
+
+.kg-bookmark-card a:hover {
+	color: black !important;
+}
+
+
+.kg-bookmark-content {    
+	flex-basis: 0;
+	flex-grow: 999;
+	min-width: 50%;
+	padding: 20px;
+	font-size: 1rem;
+	line-height: 1;
+	text-align: left;
+}
+
+.kg-bookmark-title {
+	font-weight: 600;
+	font-size: 1rem;
+}
+
+.kg-bookmark-description,
+.kg-bookmark-metadata {
+	margin-top: 12px;
+}
+
+.kg-bookmark-thumbnail {
+	flex-basis: 15rem;
+	flex-grow: 1;
+}
+
+
+.kg-bookmark-thumbnail img {
+	vertical-align: bottom;
+	object-fit: cover;
+	width: 100%;
+}
```

```suggestion
	height:100%; 
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    object-fit: cover;
    border-radius: 0 3px 3px 0;
}
```
This (with other css) will make sure the thumbnail is stretched to cover the full height of the box.